routes/resourcePages.js

```diff
@@ -97,6 +107,13 @@ async function deleteResourcePage(req, res, next) {
   IsomerFile.setFileType(resourcePageType)
   await IsomerFile.delete(pageName, sha)
 
+  // Check if collection has any pages left, and delete if none left
+  const resources = await IsomerFile.list()
+  if (_.isEmpty(resources)) {
+    const IsomerResource = new Resource(accessToken, siteName)
+    await IsomerResource.delete(resourceRoomName, resourceName)
+  }
+
```

Comment:
Instead of checking if the collection has any pages left, would it be feasible to check if there's only one page left and to delete the entire category if that's the case?

Reply:
Makes sense, resolved in https://github.com/isomerpages/isomercms-backend/pull/75/commits/c11d9fa80cbad6862658d990416b5dbf10ad3930